user_home/templates/tier1_homepage.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block menu %}
+
+<li class="list-group-item list-group-item-light menu" id="fundTransfer">
+  <a href="http://{{ request.get_host }}/internal_user/initFundDeposit">Initiate fund deposit</a> </li>
+<li class="list-group-item list-group-item-light menu" id="issueCheque">
+  <a href="http://{{ request.get_host }}/internal_user/initIssueCheque">Issue cashier cheque</a> </li>
```

Comment:
for both tier 1 and tier 2, have one more option named "Pending requests" that links to "internal_user/"
This gives list of all applicable requests such as transactions, profile update and so on.

Reply:
Pending requests is not available in urls.py Ill add place holder, people have to update it later accordingly